assets/config/bootstrap-variables.scss

```diff
@@ -244,7 +244,7 @@ $font-weight-base:            $font-weight-normal !default;
 $line-height-base:            1.47 !default;
 
 $h1-font-size:                $font-size-base * 3.17 !default;
-$h2-font-size:                $font-size-base * 2.47 !default;
+$h2-font-size:                $font-size-base * 2.33 !default;
```

Comment:
It's not what is writen in the `docs/typography.html`and I just saw that Guillaume update everything on [InVision](https://projects.invisionapp.com/share/N4FDSKWX8ZB#/screens/274002805) 😭 

I will upload those sizes outside this pull-request 😉

Reply:
*update : I juste update all the font-sizes in `dev` (9303402a44f2bf37dfd00a0c385756099a2e9940)